packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -198,18 +200,27 @@ impl<'a, T> HookContext<'a, T> {
     /// # #![cfg_attr(not(nightly), allow(dead_code, unused_variables, unused_imports))]
     /// use std::io::ErrorKind;
     ///
-    /// use error_stack::{fmt, fmt::Emit, Report};
+    /// use error_stack::{fmt::{self, Emit}, Report};
     ///
     /// struct Value(u64);
     ///
     /// Report::install_debug_hook_fallback(|frame, ctx| {
-    ///     fmt::builtin_debug_hook_fallback(frame, ctx).or_else(|| frame.is::<Value>().then(|| {
-    ///         let ctx = ctx.cast::<u64>();
-    ///         Emit::next(format!("{} (Value)", ctx.increment()))
-    ///     }))
+    ///     let builtin = fmt::builtin_debug_hook_fallback(frame, ctx);
+    ///
+    ///     if builtin.is_empty() {
+    ///         frame
+    ///             .is::<Value>()
+    ///             .then(|| {
+    ///                 let ctx = ctx.cast::<u64>();
+    ///                 vec![Emit::next(format!("{} (Value)", ctx.increment()))]
+    ///             })
+    ///             .unwrap_or_default()
+    ///     } else {
+    ///         builtin
+    ///     }
     /// });
     ///
-    /// Report::install_debug_hook::<u64>(|_, ctx| Emit::next(format!("{}", ctx.increment())));
+    /// Report::install_debug_hook::<u64>(|_, ctx| vec![Emit::next(format!("{}", ctx.increment()))]);
```

Comment:
Replace `u64` in the example with something meaningful instead, otherwise we teach bad practices.

Reply:
fixed in [4892398](https://github.com/hashintel/hash/pull/976/commits/4892398d338cb681fe7bbd5443b584fdc6383fad)